_pkgdown.yml

```diff
@@ -57,3 +57,9 @@ reference:
 - title: Package global state
   contents:
   - clear_cache
+
+authors:
+  Ramiro Magno:
+    href: https://www.pattern.institute/team/rmagno/
+  Pattern Institute:
+    href: https://www.pattern.institute
```

Comment:
```suggestion
    href: https://www.pattern.institute
  Edgar Manukyan:
    https://www.linkedin.com/in/edgar-manukyan-20987927
```

Reply:
I appended my profile in Edgar's comment above. 😸